docs/flows.md

```diff
@@ -98,7 +103,7 @@ If the flow does not implement streaming `streamFlow` will behave identically to
 You can use the CLI to stream flows as well:
 
 ```posix-terminal
-genkit flow:run jokeFlow '"banana"' -s
+genkit flow:run streamer '"banana"' -s
```

Comment:
`streamer '"banana"'`? 

This looks off. should this be: `genkit flow:run menuSuggestionFlow '"French"' -s`

Reply:
I actually didn't update the streaming example above since we don't have a menu-example for streaming yet.

Here's the
<img width="558" alt="Screenshot 2024-05-09 at 3 36 05 PM" src="https://github.com/firebase/genkit/assets/401051/14771d03-1f73-4e8c-92d9-c8a93ff6042b">
 sample code it's referenceing.

Reply:
Oh - no worries. 2 things then...

- Banana is not a valid input for the streamer flow you cited (it expects a number)
- I am primarily just noticing that the flow (sorry no pun intended) of the docs was to simply enable streaming (-s) for the previous flow (jokeFlow). So we aught to do the same in this snippet? Or if you really wanted, menuSuggestionFlowStreaming or something like that? But I think the important thing being documented here is the CLI capability (`-s`).

Reply:
Aha, i didn't see the sample code immediately above . :)

Discussed offline - we are waiting for waiting for better sample code to incorporate that is on theme.

Reply:
Took a pass at changing this so it's menu suggestion themed.